app/Http/Middleware/TrackRequestMiddleware.php

```diff
@@ -52,7 +52,7 @@ public function handle(Request $request, Closure $next)
         }
         try {
             // generating dynamic id for span with configurable prefix
-            $spanId = env('TRACE_SPAN_PREFIX', 'SPAN') . '_' . Str::uuid();
+            $spanId = env('OTEL_TRACE_SPAN_PREFIX', 'SPAN') . '_' . Str::uuid();
```

Comment:
@matiasperrone-exo this middleware should be refactored
its still creating a new span id ( should be attaching the existent one )
and should be addeding CF ray id to the context ( if its present at headers )

Reply:
also this middleware should be added using the already defined feature flag